test/packages/parallel/gcp/data_stream/compute/_dev/deploy/tf/vars.tf

```diff
@@ -0,0 +1,16 @@
+variable "TEST_RUN_ID" {
+  default = "detached"
+}
+
+variable "gcp_project_id" {
+  type = string
+}
+
+variable "zone" {
+  type = string
+  // NOTE: if you change this value you **must** change it also for test
```

Comment:
if it possible to read it from the `env.yml` config?

Reply:
We can pass it like the project id with `TF_VAR_zone`, but I would prefer having a default value so if someone runs it locally the value is already present.

Reply:
You can define default variables like [this](https://github.com/elastic/elastic-package/blob/main/test/packages/parallel/aws/data_stream/ec2_metrics/_dev/deploy/tf/env.yml#L9).